config.go

```diff
@@ -175,15 +179,30 @@ func stringsToStyle(raw []string) *Style {
 		bg: termbox.ColorDefault,
 	}
 
+	reFg := regexp.MustCompile("^\\d{1,3}$")
+	reBg := regexp.MustCompile("^on_(\\d{1,3})$")
+
 	for _, s := range raw {
 		fg, ok := stringToFg[s]
 		if ok {
 			style.fg = fg
+		} else {
+			if matchFg := reFg.FindString(s); matchFg != "" {
+				if fg, err := strconv.ParseUint(matchFg, 10, 8); err == nil {
+					style.fg = termbox.Attribute(fg+1)
+				}
+			}
 		}
 
 		bg, ok := stringToBg[s]
 		if ok {
 			style.bg = bg
+		} else {
+			if matchesBg := reBg.FindStringSubmatch(s); matchesBg != nil {
```

Comment:
@syohex @lestrrat There is no problem. Whether the block is run or not, termbox handles correctly depending on `termbox.OutputMode`.

Anyway, I think that checking `Use256Color` in `stringToStyle` is difficult because `Use256Color` and `Style` are in same JSON.